tabcmd/commands/auth/session.py

```diff
@@ -1,10 +1,12 @@
 import getpass
-from ..constants import Constants
+from tabcmd.commands.constants import Constants
 import tableauserverclient as TSC
 from tabcmd.execution.logger_config import log
 import json
 import os
 from ..commands import Commands
```

Comment:
Nit: Not sure if it will work but I wonder if this can also be "from tabcmd.commands.commands import Commands "?